docs/tutorials/getting_started/laziness.md

```diff
@@ -20,11 +20,6 @@ This tutorial will show both you how to create `DatasetReader`s
 that allow this lazy behavior, and how to handle this laziness
```

Comment:
Unrelated to this file, but it's the closest file I can comment on: why have `docs/api/.gitkeep`?  You want the directory to be around in some script that generates the docs?  Why not just create that directory in the script?